README.md

```diff
@@ -12,35 +12,70 @@ You can perform unit tests with `go test` or `make test`. Enable verbose logging
-| `logger`                       | `string`   | UDP host and port that `sshmux` send log messages to.                                                                               | `false`  | `"127.0.0.1:5556"`                 |
-| `proxy-protocol-allowed-cidrs` | `[]string` | CIDRs from which [PROXY protocol](https://www.haproxy.com/blog/use-the-proxy-protocol-to-preserve-a-clients-ip-address) is allowed. | `false`  | `["127.0.0.22/32"]`                |
-
-### Advanced Config
-
-The table below shows extra options for `sshmux`, mainly for authentication with Vlab backends:
-
-| Key                        | Type       | Description                                                                | Example                      |
-| -------------------------- | ---------- | -------------------------------------------------------------------------- | ---------------------------- |
-| `recovery-token`           | `string`   | Token used to authenticate with the recovery backend. Defaults to `token`. | `"long-and-random-token"`    |
-| `recovery-server`          | `string`   | SSH host and port of the recovery server.                                  | `"172.30.0.101:2222"`        |
-| `recovery-username`        | `[]string` | Usernames dedicated to the recovery server.                                | `["recovery", "console"]`    |
-| `all-username-nopassword`  | `bool`     | If set to `true`, no users will be asked for UNIX password.                | `true`                       |
-| `username-nopassword`      | `[]string` | Usernames that won't be asked for UNIX password.                           | `["vlab", "ubuntu", "root"]` |
-| `invalid-username`         | `[]string` | Usernames that are known to be invalid.                                    | `["user"]`                   |
-| `invalid-username-message` | `string`   | Message to display when the requested username is invalid.                 | `"Invalid username %s."`     |
-
-All of these options can be omitted, if the corresponding feature is not intended to be used.
+`sshmux` requires a TOML configuration file to start up. By default it will look at `/etc/sshmux/config.toml`, but you can also specify a custom configuration by passing `-c path/to/config.toml` in the command line arguments. An [example](etc/config.example.toml) file is provided.
+
+The sections below will introduce available options for `sshmux`:
+
+### General Settings
+
+General settings configure the `sshmux` service. They are top-level settings in the TOML file.
+
+| Key       | Type     | Description                                     | Required | Example          |
+| --------- | -------- | ----------------------------------------------- | -------- | ---------------- |
+| `address` | `string` | TCP host and port that `sshmux` will listen on. | `true`   | `"0.0.0.0:8022"` |
```

Comment:
Just personal preference: Since this Markdown table is intended for humans, the `true` / `false` content for the "Required" column might be better written as "Yes" / "No" (or "Yes" / empty).